src/pages/certificates/BrowserImport.tsx

```diff
@@ -132,6 +152,33 @@ const BrowserImport: FC = () => {
             </ul>
           </div>
         )}
+
+        {activeTab === MACOS && (
+          <div role="tabpanel" aria-label="safari macos">
+            <ul className="p-list--divided u-no-margin--bottom">
+              <li className="p-list__item">
+                The certificate must be protected by password. An empty password
+                will fail to be imported on macOS.
+              </li>
+              {downloadPfx}
+              <li className="p-list__item">
+                Start the Keychain Access app on your Mac, select the login
+                keychain.
+              </li>
+              <li className="p-list__item">
+                Drag the certificate file onto the Keychain Access app.
+              </li>
+              <li className="p-list__item">
+                If you’re asked to provide a name and password, type the name
```

Comment:
Let's consider this formal writing and do without contractions:
```suggestion
                If you are asked to provide a name and password, type the name
```